docs/content/configuration/prefill-disk-cache.md

```diff
@@ -28,7 +30,7 @@ $ go get github.com/plexsystems/pacmod
 
 This command will install the `pacmod` binary to your `$GOPATH/bin/pacmod` directory, so make sure that is in your `$PATH`.
 
-# Next, run `pacmod` to create assets
+### Next, run `pacmod` to create assets
```

Comment:
Small nit, would you be able to change this to a bold'ed text rather than a headline for a bit better readability.